charts/stable/common/templates/_deployment.tpl

```diff
@@ -42,9 +42,9 @@ spec:
       {{- include "common.labels.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
+      {{ if .Values.podAnnotations }}
```

Comment:
```suggestion
      {{- if .Values.podAnnotations }}
```

Reply:
I think we need to keep the whitespace, because I think otherwise it would render as `metadata:annotations:`